Book/php7/internal_types/functions.rst

```diff
@@ -1,4 +1,15 @@
-Functions: zend_function
+Functions
 ========================
 
-Here we detail how functions work in PHP.
+The body of PHP functions are represented with the ``zend_function`` structure.
+However, handling them is rarely done as they are solely needed for the VM.
+In general PHP ``callable`` s are what will need to be dealt with, which are represented by the pair of
+``zend_fcall_info``/``zend_fcall_info_cache`` structures.
+
+
+TODO: Detail ``zend_function``
```

Comment:
Not 100% wether we should commit with this `TODO` – might make sense, though.

Reply:
I mean, I could create a blank file and link it here, similarly to how we currently have blank pages for the HashTable, but not sure if that's that much better than the TODO

Reply:
Then just keep the TODO. :)